### PR TITLE
Draft Add virtual_dataframe package, version v0.1rc2 (need to be validated locally before to be revue)

### PR DESCRIPTION
**Pull request by pprados**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/virtual_dataframe/conda-forge.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+channels:
+  # sources selects the channels to pull packages from, in order.
+  sources:
+    - conda-forge
+    - rapidsai
```
Comment (bollwyvl) [inline recipes/virtual_dataframe/conda-forge.yml:5]: just a heads-up: this probably won't get approved

File: recipes/virtual_dataframe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+requirements:
+  build:
+    - python
+#    - python {{ PYTHON_VERSION }}
+    - pip  # Indispensable pour une installation correct avec scm
+    - setuptools # Indispensable pour une installation correct avec scm
+    - setuptools_scm # Indispensable pour une installation correct avec scm
+
+#  host:
```
Comment (bollwyvl) [inline recipes/virtual_dataframe/meta.yaml:44]: you almost certainly need a host environment

File: recipes/virtual_dataframe/conda-forge.yml (hunk 1 above)
Comment (pprados) [reply recipes/virtual_dataframe/conda-forge.yml:5]: It's not possible to have a package with a dependency from another channel ?

File: recipes/virtual_dataframe/meta.yaml (hunk 2 above)
Comment (pprados) [reply recipes/virtual_dataframe/meta.yaml:44]: How I can have a host environement ?
I will try without the dependencies of setuptools_scm (I already tested without result).

**Pull request closed without merging**